extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -151,10 +151,13 @@ interface SetStateMsg {
 export interface UserSettings {
   /** Whether to display links to the dataflow models that generated particular nodes in a flow path. */
   shouldShowProvenance: boolean;
+  /** Whether to display the "View Autofixes" button. */
+  shouldShowViewAutofixesBtn: boolean;
```

Comment:
Let me know if I should make `shouldShowViewAutofixesBtn` and `shouldShowProvenance` optional. It feels a bit ugly that I am setting these values for views where they are not relevant.